### ISSUE TITLE
[Feature Request]: SOLECTRUS-Importer über Oberfläche nutzbar machen

### ISSUE THREAD

**Issue by noffycws** [solectrus/solectrus#1869]
Da es nun eher einer neuen Idee / einem Feature Request geht, mal die Antwort auf #1132 hier fortgeführt:

Über den SOLECTRUS-Importer bin ich mittlerweile auch "gestolpert". Leider wird der in der Beschreibung nicht erwähnt - oder ich hab es einfach überlesen.

Vielleicht ist es aber möglich in der Oberfläche einen Punkt einzubauen, mit dem man die CSV'en einladen kann (ggf. als ZIP, welches dann entpackt wird) und man von dort aus den Import starten kann (inkl Restart von redis?). Ist für Nicht-Techies vielleicht eine Erleichtung und ein einfacher Weg die Daten einzubinden.

Soweit meine Idee für den Feature Request.

**Comment by ledermann** [comment]
Ja, der SENEC-Importer wurde nicht erwähnt, habe ich nun ergänzt, danke für den Hinweis.

Ein Upload der CSV-Dateien über den Import wäre zweifellos eine deutliche Verbesserung der UX. Die Realisierung ist aber auch aufwändig, daher habe ich mir da bislang gespart. Mal sehen, ob ich das später mal angehe.
